draft-edm-protocol-greasing.md

```diff
@@ -100,12 +100,16 @@ Implementations that receive grease values are required to ignore them. More
 background to this approach is given in {{Section 3.3 of ?VIABILITY=RFC9170}}.
 This section provides concrete suggestions for its usage.
 
+## Don't Special-Case Grease Values
+
```

Comment:
Is this clearer as: Don't make Grease Values a Special Case?

Reply:
I took the suggestion and mashed it up a little